### ISSUE TITLE
several errors: mainly impossible editing some columns

### ISSUE THREAD

**Issue by DoctorSubtilis** [phpmyadmin/phpmyadmin#17557]
### To Reproduce

1. Go to 'a table'
2. Click on 'edit' a row
3. Do some modification
4. See error:  `Warning in ./libraries/classes/InsertEdit.php#1765
 implode(): Invalid arguments passed

Backtrace
```
InsertEdit.php#1765: implode(
string ',',
string 'pro_doct',
)
ReplaceController.php#312: PhpMyAdmin\InsertEdit->getCurrentValueForDifferentTypes(
boolean false,
string '2ab6240bef8140b28277f912b86342bf',
array,
string '\'\'',
array,
integer 0,
array,
array,
array,
boolean false,
boolean true,
string '`bibliografie`.`ID` = 2',
string 'bibliografie',
array,
)
Routing.php#192: PhpMyAdmin\Controllers\Table\ReplaceController->__invoke(
,
array,
)
index.php#43: PhpMyAdmin\Routing::callControllerForRoute(
,
string '/table/replace',
,
,
)
```
5. no modification made

In these same day I have other errors (probably linked):  

* impossible copying a table from database to another (no error messages) 
* impossible create view (`create view`, after a suffessfull query does't work; no error messages)

### Expected behavior

The usual behavior: modifying items and copy tables, and create views without errors.

### Server configuration

- Operating system: KDE Neon
- Apache/2.4.46 (Unix) OpenSSL/1.1.1i PHP/7.4.14 mod_perl/2.0.11 Perl/v5.32.0
- Database client version: libmysql - mysqlnd 7.4.14
- PHP extension: mysqli Documentation curl Documentation mbstring Documentation
- PHP version: 7.4.14

### Client configuration

- Browser: any
- Operating system: KDE Neon

**Comment by williamdes** [comment]
Hi @DoctorSubtilis 
What phpMyAdmin version are you using ?

**Comment by DoctorSubtilis** [comment]
> Hi @DoctorSubtilis What phpMyAdmin version are you using ?

Version information: 5.2.0 (up to date)

**Comment by williamdes** [comment]
Thank you !
https://github.com/phpmyadmin/phpmyadmin/blob/848c4ef172adf463f9032eca71cb33559c4836fa/libraries/classes/InsertEdit.php#L1765

Can you provide the table structure ?
I think you have a field that uses a 'SET' type

**Comment by williamdes** [comment]
> 

No need to remove the `SET` type, this is a bug. I am trying to get some example data to be sure we cover the case you are seeing. Do you have some specific config ?

**Comment by DoctorSubtilis** [comment]
What do you mean? I don't think so....

**Comment by williamdes** [comment]
> What do you mean? I don't think so....

I mean that you should not alter your structure or change your data because of phpMyAdmin
Is there is a bug with your data or structure we need to fix it on our side

**Comment by Mutagenic** [comment]
Hi, I also have the same issue but do not have a SET field in out table but an ENUM field, if that helps somehow. Thanks!

**Comment by ethaniel** [comment]
I've noticed this error too, however the PHP error about "InsertEdit.php" and "implode()" appears and disappears on the screen so quickly, that I had to use Chrome Developer Tools to see the message in the Ajax log (and only then I could find this issue).

I suspect many other people are having this problem too, but can't read the error message in time, so this might be worth looking into as well.

**Comment by OliverTUBAF** [comment]
I'm also affected by this bug. What I can't do anymore is copying/duplicating a row of data inside the same table. I also got a SET field any many ENUMs.

**Comment by williamdes** [comment]
Could someone test the fix in #17650 and let us know if it fixes the issue described here ?
I think it should, but just to be sure

**Comment by OliverTUBAF** [comment]
> Could someone test the fix in #17650 and let us know if it fixes the issue described here ? I think it should, but just to be sure

Although almost one month has passed and it is already closed: I can confirm that this the change in #17650 fixed the issue for me.
